### ISSUE TITLE
Output file Is almost 39 hours long

### ISSUE THREAD

**Issue by Emersont1** [sandreas/m4b-tool#120]
```
$ m4b-tool merge -v Doctor\ Who:\ Shada  --jobs=4 --output-file="Doctor Who: Shada.m4b" --
determine highest available audio codec
reading metadata and streaminfo for file /mnt/Doctor Who: Shada/Disc 1 - 01 - Prelude.flac
+
searching for cover in Doctor Who: Shada
using cover  Doctor Who: Shada/cover.jpg
searching for description.txt in Doctor Who: Shada
success: found description.txt for import
skip cover extraction, file Doctor Who: Shada/cover.jpg already exists - use --force to overwrite
preparing conversion with 4 simultaneous jobs, please wait...
   0 remaining /   37 total, preparing next task |
using cover  Doctor Who: Shada/cover.jpg
searching for chapters.txt in Doctor Who: Shada
file chapters.txt not found or too big
rebuilding chapters from converted files title tags
merging Doctor Who: Shada-tmpfiles//tmp_Doctor Who: Shada.m4b, this can take a while
+++
check for sorting support in mp4tags
supported, proceeding...
tagging file Doctor Who: Shada-tmpfiles//tmp_Doctor Who: Shada.m4b
adding cover Doctor Who: Shada/cover.jpg to Doctor Who: Shada-tmpfiles//tmp_Doctor Who: Shada.m4b
importing chapters for Doctor Who: Shada-tmpfiles//tmp_Doctor Who: Shada.m4b

```

The output is 38:47 long, where all the chapters are at the beginning

![image](https://user-images.githubusercontent.com/23140112/106200970-b217b000-61af-11eb-865b-668712b9b64d.png)

**Comment by sandreas** [comment]
Are you using the latest pre-release? If not, try it, this should be fixed here.

**Comment by Emersont1** [comment]
I built from `master`

**Comment by sandreas** [comment]
Ok great, thanks for the quick feedback, I'll take a look in the next days.

**Comment by sandreas** [comment]
Sorry for the delay... Should now be fixed, see #135
